decidim-admin/app/commands/decidim/admin/destroy_participatory_process_step.rb

```diff
@@ -30,7 +30,7 @@ def call
 
       def reorder_steps
         ReorderParticipatoryProcessSteps
-          .new(@participatory_process.steps, @participatory_process.steps.map(&:id))
+          .new(@participatory_process.steps.reload, @participatory_process.steps.map(&:id))
```

Comment:
We need to reorder to avoid this: https://github.com/AjuntamentdeBarcelona/decidim/pull/990

Working on the other issue!